src/mono/msbuild/WasmAppBuilder/WasmAppBuilder.csproj

```
@@ -0,0 +1,18 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <OutputType>Library</OutputType>
+    <OutputPath>bin</OutputPath>
+    <TargetFramework>$(NetCoreAppCurrent)</TargetFramework>
```

There was a recent change to move the AppBuilders to artifacts/ in https://github.com/dotnet/runtime/commit/741e9054d25adc8a93fd2695b0848adcecaaffb0 so you can remove this:

```suggestion
    <OutputPath>bin</OutputPath>
    <TargetFramework>$(NetCoreAppCurrent)</TargetFramework>
```

You'll need to change the paths where it is referenced.